backend/src/zac/contrib/dowc/permissions.py

```diff
@@ -1,10 +1,8 @@
-from rest_framework import permissions
-from rest_framework.request import Request
-from rest_framework.views import APIView
-
+from zac.accounts.constants import PermissionObjectType
+from zac.api.permissions import DefinitionBasePermission
 from zac.core.permissions import zaken_download_documents
 
 
-class CanOpenDocuments(permissions.BasePermission):
-    def has_permission(self, request: Request, view: APIView) -> bool:
-        return request.user.has_perm(zaken_download_documents.name)
+class CanOpenDocuments(DefinitionBasePermission):
+    permission = zaken_download_documents.name
```

Comment:
according to the type hint, this should be:

```suggestion
    permission = zaken_download_documents
```